### PR TITLE
first media queries for the ohmyfood main page

### PR DESCRIPTION
**Pull request by Dylanm13**
I made a first try for mobile version, waiting for your feedback ! By the way, I forgot to create a new branch for this, sorry for that i'll be carefull next time ! When I'll pull this branch i'll think to create new ones

### CODE REVIEW

File: css/explore.css [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-  background: linear-gradient(#FF79DA, #9356DC);
-  color: white;
-  border-radius: 40px;
-  border: none;
-  font-size: 2rem;
-  height: 5rem;
-  padding: 0px 35px;
-  box-shadow: 2px 2px 5px grey;
-}
+/* Error: Undefined variable.
```
Comment (benjaminW78) [inline css/explore.css]: so what do we do ? do you keep that like that even if it's a useless comment ?

File: sass/explore.scss [hunk 2]
```diff
@@ -1,14 +1,17 @@
-$color-primary:#9356DC;
-$color-secondary:#FF79DA;
-
 .explore {
```
Comment (benjaminW78) [inline sass/explore.scss:3]: Nice file structure with SASS/ folder and css for computed files.

File: sass/style.scss [hunk 3]
```diff
@@ -1,3 +1,10 @@
+$color-primary:#9356DC;
```
Comment (benjaminW78) [inline sass/style.scss]: does it work as you expected ?

Comment (benjaminW78) [reply sass/style.scss]: + why is this file called style.scss and not colors or something related to what i can see here and bellow.

**Review by benjaminW78 — CHANGES_REQUESTED** [review]

File: css/explore.css (hunk 1 above)
Comment (Dylanm13) [reply css/explore.css]: I'll make sure to delete this comment

File: sass/explore.scss (hunk 2 above)
Comment (Dylanm13) [reply sass/explore.scss:3]: Thanks !

File: sass/style.scss (hunk 3 above)
Comment (Dylanm13) [reply sass/style.scss]: I'll create a color.scss file and keep the police in style.scss

**Review by benjaminW78 — APPROVED** [review]

**Pull request merged**